docs/tutorialkit.dev/src/content/docs/reference/configuration.mdx

```diff
@@ -70,12 +70,53 @@ type I18nText = {
   startWebContainerText?: string,
 
   /**
-   * Text shown on the call to action button to start webcontainer when boot was blocked
-   * due to memory restrictions.
```

Comment:
Oh good catch! This comment is meant to be on top of `startWebContainerText?: string,`

Reply:
So do we also need to update `i18n.ts`?

https://github.com/stackblitz/tutorialkit/blob/e45aa1523f82f71a406bdc63c98d7a0a7e277371/packages/types/src/schemas/i18n.ts#L25-L33

Reply:
Oh good catch! Yes we should! :smiley:

Reply:
done